lambda-buffers-compiler/src/Orphan/Text.hs

```diff
@@ -0,0 +1,17 @@
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Orphan.Text () where
```

Comment:
Why not https://hackage.haskell.org/package/quickcheck-instances? Also base types are almost all sized afaik, did you run into some problems?

Reply:
Yes - my tests were OOM'ing - see the conversation on slack. The sized instances were what made it stop.